src/PKSim.Presentation/Presenters/Main/SimulationExplorerPresenter.cs

```diff
@@ -316,6 +321,32 @@ private static bool shouldIgnoreDoubleClick(object tag)
          return tag.IsAnImplementationOf<UsedBuildingBlock>() || tag.IsAnImplementationOf<UsedObservedData>();
       }
 
+      public override IEnumerable<IMenuBarItem> AllCustomMenuItemsFor(ClassificationNode classificationNode)
+      {
```

Comment:
This is it: 
On custom items we

1. Save the localization of the classification
2. Add the custom menus

It was easy to add for PU and SA even it it was not specified.